hipe4ml/tree_handler.py

```diff
@@ -127,10 +126,9 @@ def get_handler_from_large_file(self, file_name, tree_name, model_handler=None,
         self._files = file_name
         self._tree = tree_name
 
-        executor = ThreadPoolExecutor(
-            max_workers) if max_workers is not -1 else None
-        iterator = uproot.pandas.iterate(
-            file_name, tree_name, executor=executor)
+        executor = ThreadPoolExecutor(max_workers) if max_workers != -1 else None
+        iterator = uproot.iterate(f'{file_name}:{tree_name}', library='pd', decompression_executor=executor,
```

Comment:
The new version of uproot.iterate does not support reading lists of files. It would be useful to add this option as it is done for TreeHandler.__init__
https://uproot.readthedocs.io/en/latest/uproot.behaviors.TBranch.iterate.html

Reply:
It does, a small change was needed although. I added also some tests for the loading